bedrock/firefox/templates/firefox/whatsnew/includes/fx83/person2.svg

```diff
@@ -0,0 +1 @@
8 9.697 15.544c2.923 4.064 10.125 10.268 10.125 10.268M46.745 168.967s8.128-.214 12.407.214c4.278.428 12.62 1.782 12.62 1.782M44.25 179.092s11.836-.356 16.328-.356c4.492 0 9.34 1.14 9.34 1.14M44.035 193.638s16.543-5.99 26.24-7.986" stroke="#FFC0D1" stroke-width="1.5" stroke-miterlimit="10"/><path d="M146.783 141.943s-6.774 11.052-21.961 17.897c-15.188 6.774-29.948 10.696-51.695 9.626M150.99 154.778s-9.483 8.77-22.888 16.542M154.769 164.689s-2.923 4.278-10.909 12.05c-7.986 7.772-13.405 11.266-13.405 11.266" stroke="#FFC0D1" stroke-width="1.5" stroke-miterlimit="10"/><path d="M74.767 165.758l-4.136 8.486" stroke="#000" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/><path d="M82.681 132.246s5.063 10.695 13.62 11.622c8.556.999 16.542-3.28 18.467-7.986 1.069-2.567 1.355-4.135 1.426-4.991 0-.428.356-.713.784-.713.285 0 .499-.214.927-.285 1.783-.214 3.565-2.71 2.71-6.774 0 0-6.774-1.925-11.837-5.847-5.062-3.85-8.128-12.264-8.128-12.264s-6.417 3.494-9.483 12.05c-3.993 10.838-8.486 15.188-8.486 15.188z" fill="#fff"/><path d="M100.864 135.669s-.785-1.355-2.353-1.284c-1.569.071-2.353 2.71-1.925 4.207.356 1.283.713 2.353 2.852 2.496 2.139.214 4.349-1.926 3.993-4.065-.357-2.139-2.567-1.354-2.567-1.354z" fill="#F35"/><path d="M92.593 131.818s2.067 6.204 12.692 6.988M96.514 127.397s0 3.138 1.569 3.851c1.569.713 1.782-.785 2.567-.499.784.285 0 2.067 1.569 2.353 1.568.285 2.495-3.138 2.495-3.138" stroke="#000" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/><path d="M109.491 123.904a1.213 1.213 0 100-2.426 1.213 1.213 0 000 2.426zM90.168 123.904a1.213 1.213 0 100-2.425 1.213 1.213 0 000 2.425z" fill="#000"/><path d="M130.098 206.615l11.694-35.509c.713-2.282-1.07-4.563-3.423-4.349l-69.734 6.132c-1.783.142-2.852 1.996-2.068 3.565l14.831 32.371a3.602 3.602 0 003.28 2.139l37.648.999c3.494.285 6.703-1.925 7.772-5.348z" fill="#fff"/><path d="M104.5 194.066c2.639 0 4.778-1.852 4.778-4.136s-2.139-4.135-4.778-4.135c-2.638 0-4.777 1.851-4.777 4.135s2.139 4.136 4.777 4.136z" stroke="#000" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/><path d="M47.386 155.134s11.48-2.353 17.113-6.203c5.633-3.851 17.256-15.33 18.254-16.685.998-1.355-8.556 1.355-9.127 1.569-.57.214-19.252 12.62-19.252 12.62l-6.988 8.699z" fill="#FCB643"/><path d="M42.68 156.132c.57 0 16.543-2.923 26.026-10.481s17.47-16.685 19.822-22.532c2.353-5.847 5.277-12.621 7.558-14.974 2.282-2.353 4.493-3.137 4.493-3.137M106.212 101.586s4.492-.571 6.773 2.139c2.282 2.709 5.277 8.913 6.418 12.05 1.14 3.137 2.923 6.631 7.415 7.772" stroke="#000" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/><path d="M96.728 103.511s-4.492.998-9.126 6.417c-3.138 3.708-12.407 20.036-20.037 25.812-7.63 5.775-13.619 9.911-18.824 11.266-5.276 1.354-13.048 0-13.048 0" stroke="#000" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
The tool you used to minimize these removed the view box. That's okay for WNP since they're Firefox only but you should adjust the settings so it doesn't do this if the image is going to be displayed on other pages.

Reply:
I'll reformat these to keep the view box.

Reply:
Done!